### DISCUSSION TITLE
How do I build/Integrate nanobind?

### DISCUSSION THREAD

**Discussion by OmerUygurOzer** [wjakob/nanobind#110] [Q&A]
I see no documentation on how to build and integrate nanobind. I tried building it using the top level cmakelist file but it seems like it's only for tests? Even then I actually get the following error:

`CMake Error at tests/CMakeLists.txt:62 (add_custom_command):
  add_custom_command called with OUTPUT containing a "<".  This character is
  not allowed.
`

So how do I build it into a .lib and/or a single header maybe? Not sure exactly how to integrate.

**Comment by wjakob** [comment]
Please take a look at the official example: https://github.com/wjakob/nanobind_example. Not sure what went wrong with your error message, perhaps a very old CMake version?
